Integrations/integration-RsaNetwitnessSecurityAnalytics.yml

```diff
@@ -1081,57 +1297,60 @@ script:
         case 'nw-login':
             results = sessionId;
             break;
-        case 'nw-list-incidents':
+        case 'netwitness-im-list-incidents':
             var incidents = listIncidents(sessionId, args, incidentManagementId);
-            results = { incidents: incidents };
+            results = buildRetValListIncidents(incidents);
             break;
-        case 'nw-get-incident-details':
+        case 'netwitness-im-get-incident-details':
             var incident = getIncidentById(sessionId, args, incidentManagementId);
-            results = { incident: incident };
+            results = buildRetValGetIncidentDetails(incident);
             break;
-        case 'nw-get-components':
+        case 'netwitness-im-get-components':
             var components = getComponents(sessionId, args);
-            results = { components: components };
+            results = buildRetValGetComponents(components);
             break;
-        case 'nw-get-events':
+        case 'netwitness-im-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = buildRetValGetEvents(events);
             break;
-        case 'nw-get-event-details':
-            results = getEventDetails(sessionId, args);
+        case 'netwitness-im-get-event-details':
+            event = getEventDetails(sessionId, args);
+            results = buildRetValGetEventDetails(event);
             break;
-        case 'nw-get-alerts':
+        case 'netwitness-im-get-alerts':
             var alerts = filterAlerts(sessionId, args);
-            results = { alerts: alerts };
+            results = buildRetValGetAlerts(alerts);
             break;
-        case 'nw-get-alert-details':
+        case 'netwitness-im-get-alert-details':
             var alert = getAlertById(sessionId, args);
-            results = { alert: alert };
+            results = buildRetValGetAlert(alert);
             break;
-        case 'nw-get-alert-original':
+        case 'netwitness-im-get-alert-original':
             var alert = getOriginalAlertById(sessionId, args);
```

Comment:
I know this is not related to your changes. But please fix this small issue too.
We get this error when such alert not exists
![image](https://user-images.githubusercontent.com/7270217/28249399-8bc5f2ec-6a5d-11e7-84b7-ccdfdf53859c.png)